models/ed/tests/testthat/test.read_E_files.R

```diff
@@ -1,19 +1,27 @@
+outdir <- tempdir()
```

Comment:
As above, `tempdir()` isn't cleaned up until the R session exits, so this could leave test outputs behind to interfere with later runs of the tests in the same session. I recommend directing all outputs into a dir that's explicitly cleaned up at the end of the test.